.github/workflows/ci.yml

```diff
@@ -64,16 +64,24 @@ jobs:
     name: Release build
     runs-on: ubuntu-latest
     container:
-      image: registry.gitlab.com/finestructure/spi-base:1.1.1
+      image: registry.gitlab.com/finestructure/spi-base:1.2.0
       options: --privileged
     steps:
       - name: GH Runner bug workaround
         run: sysctl -w vm.mmap_rnd_bits=28
       - name: Checkout code
         uses: actions/checkout@v4
         with: { 'fetch-depth': 0 }
-      - name: Build release
-        run: swift build -c release --static-swift-stdlib -Xlinker -ljemalloc
+      - name: Test release
+        run: |
+          # Since we're only pinging the version api endpoint, no database is needed and
+          # we can just copy the development template to ensure the db env variables are set.
+          cp .env.development.template .env.development
+          swift build -c release --static-swift-stdlib -Xlinker -ljemalloc
+          $(swift build --show-bin-path -c release)/Run serve --port 8080 --hostname 0.0.0.0 &
+          sleep 10
+          echo Probing api/version...
+          bash -c '[ "$(curl -sL -w "%{http_code}" -o /dev/null http://127.0.0.1:8080/api/version)" == "200" ] || (echo Connection failed ; exit 1)'
```

Comment:
This is a new smoke test where we launch the release build and probe it via curl.

I've tested this with a debug build (which is not broken) here: https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/actions/runs/14441053515/job/40491081845?pr=3761